apps/web/app/s/[surveyId]/components/LinkSurvey.tsx

```diff
@@ -181,7 +181,9 @@ export default function LinkSurvey({
 
   return (
     <div className="flex h-screen items-center justify-center">
-      {!determineStyling().isLogoHidden && product.logo?.url && <ClientLogo product={product} />}
+      {product.logo?.url && (
```

Comment:
we have one check here anyways so let's keep all the "can and should we show the logo" here on this layer instead of passing it down.